tests/pipeline/test_estimate.py

```
@@ -0,0 +1,166 @@
+
+        # Load true ATE
+        true_ate = load_true_ate(data_dir)
+        if true_ate is None:
+            return False
+
+        print(f"True ATE: {true_ate:.4f}")
+
+        # Test each estimate (assuming there might be multiple rows)
+        all_passed = True
+
+        for idx, row in df.iterrows():
```

Ruff (0.11.9)

Loop control variable `idx` not used within loop body

Rename unused `idx` to `_idx`

(B007)